amp/WEB-INF/src/org/digijava/module/aim/action/CompareActivityVersions.java

```diff
@@ -365,13 +367,23 @@ public ActionForward compareAll(ActionMapping mapping, ActionForm form, HttpServ
                                     HttpServletResponse response) throws Exception {
 
         CompareActivityVersionsForm vForm = (CompareActivityVersionsForm) form;
-        vForm.populateEffectiveFilters();
+        HttpSession session = request.getSession();
+        TeamMember member  = (TeamMember) session.getAttribute("currentMember");
+        String workspace = member.getTeamName();
+        if(workspace.equals("AMP Administrator")) {
```

Comment:
Is better to check agains session atribute "ampAdmin" to check wheter is admin or not, also you should check if tm.getTeamHead() to avoid the page being accesed by non approvers/managers of the team